net/include/lute/net.h

```diff
@@ -21,4 +21,4 @@ static const luaL_Reg lib[] = {
     {nullptr, nullptr},
 };
 
-} // namespace net
+} // namespace net
```

Comment:
```suggestion
} // namespace net

```

Do not strip the final line break.